src/wallet/rpcdump.cpp

```diff
@@ -601,7 +609,7 @@ Value z_importkey(const Array& params, bool fHelp)
 
         // We want to scan for transactions and notes
         if (fRescan) {
-            pwalletMain->ScanForWalletTransactions(chainActive.Genesis(), true);
+            pwalletMain->ScanForWalletTransactions(chainActive[nRescanHeight], true);
```

Comment:
`CChain::operator[]` returns `NULL` for invalid heights, and `ScanForWalletTransactions` is a no-op for `NULL`.